velox/exec/SortBuffer.h

```diff
@@ -87,6 +89,9 @@ class SortBuffer {
   // TableWriter to indicate if this sort buffer object is under non-reclaimable
   // execution section or not.
   tsan_atomic<bool>* const nonReclaimableSection_;
+  // The flag is passed from the associated operator such as OrderBy to pass the
```

Comment:
PrefixSortConfig is a struct with 2 integers. It is not a flag. Let's not use 'flag' term to refer to it. Consider,

// Configuration settings for prefix-sort.

Or just remove the comment altogether as it doesn't seem to add any additional information.